components/browser/state/src/main/java/mozilla/components/browser/state/state/BrowserState.kt

```diff
@@ -38,6 +39,7 @@ data class BrowserState(
     val containers: Map<String, ContainerState> = emptyMap(),
     val extensions: Map<String, WebExtensionState> = emptyMap(),
     val activeWebExtensionTabId: String? = null,
+    val inProgressMediaTabId: String? = null,
```

Comment:
I don't fully understand the need for this: Isn't this already derivable from the current state, since every tab can expose whether it is playing or not? Also note that currently GV only one ever has one playing tab, but this is not something that has to be this way, there could be multiple playing tabs.

Reply:
If the app was closed in the background, when the user opens it again we'd show the last tab which had media playing allowing her to easily resume. This would be an easier way than to save/restore the media state for every tab and then find the last accessed tab with `MediaSessionState` data.
And if we'd do that we'd not know when to stop. There can be more tabs with `MediaSessionData` that we'd again show as the last tab with in progress media. But they would actually be the penultimate and so on. Whether with this property we have two states: the last tab playing media or nothing.